pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -51,6 +50,21 @@ let
       hash = "sha256-bQC0QmkLalxdj4mDEdqvvOFtNwz2T1MpTDuMXGYeQ18=";
     };
   };
+
+  # Derived from subprojects/spdlog.wrap
+  spdlog = rec {
+    version = "1.8.5";
+    src = fetchFromGitHub {
+      owner = "gabime";
+      repo = "spdlog";
+      rev = "refs/tags/v${version}";
+      sha256 = "sha256-D29jvDZQhPscaOHlrzGN1s7/mXlcsovjbqYpXd7OM50=";
+    };
+    patch = fetchurl {
+      url = "https://wrapdb.mesonbuild.com/v2/spdlog_${version}-1/get_patch";
+      sha256 = "sha256-PDjyddV5KxKGORECWUMp6YsXc3kks0T5gxKrCZKbdL4=";
+    };
+  };
```

Comment:
Wouldn't it be better to use our `spdlog.src` for this?

Reply:
I suppose we could, but I did this so it's consistent with source defined in `subprojects/spdlog.wrap` and to avoid problems if the src ever gets out of sync with the patch.